src/types/Card.ts

```diff
@@ -0,0 +1,63 @@
+export namespace Card {
+  export type Brand =
+    | 'AmericanExpress'
+    | 'DinersClub'
+    | 'Discover'
+    | 'JCB'
+    | 'MasterCard'
+    | 'UnionPay'
+    | 'Visa'
+    | 'Unknown';
+
+  export type TokenType =
+    | 'Account'
+    | 'BankAccount'
+    | 'Card'
+    | 'CvcUpdate'
+    | 'Person'
+    | 'Pii';
+
+  export interface Token {
+    id: string;
+    created: number;
+    type: TokenType;
+    used: boolean;
+    livemode: boolean;
+    card?: Params;
+    bankAccount?: BankAccount;
+  }
+
+  export interface BankAccount {
+    bankName: string;
+    accountHolderName: string;
+    accountHolderType: 'Individual' | 'Company';
+    currency: string;
+    country: string;
+    routingNumber: string;
+    status: 'Errored' | 'New' | 'Validated' | 'VerificationFailed' | 'Verified';
+  }
+
+  export interface Params {
+    country: string;
+    brand: Card.Brand;
+    currency: string;
+    expMonth: number;
+    expYear: number;
+    last4: string;
+    funding: 'Credit' | 'Debit' | 'Prepaid' | 'Unknown';
+    address: Address;
+  }
+
+  export interface Address {
+    addressCity: 'Houston';
+    addressCountry: 'US';
+    addressLine1: '1459  Circle Drive';
+    addressLine2: 'Texas';
+    addressPostalCode: '77063';
+  }
```

Comment:
These need to be types instead of demo values.

Also, for some reason I'm getting different shapes:

* iOS
```js
 "card": {
    "expYear": 2022,
    "brand": null,
    "name": null,
    "currency": null,
    "expMonth": 12,
    "last4": "3220",
    "country": "IE",
    "address": {
      "city": null,
      "country": null,
      "phone": null,
      "postalCode": null,
      "line2": null,
      "email": null,
      "line1": null,
      "name": null,
      "state": null
    },
    "funding": "Credit"
  },
  "created": 1621345575000,
  "livemode": false,
  "bankAccount": null,
  "tokenId": "tok_1IsTFTG7gfqYEkPO0FWNs04e",
  "type": null
}
```

* Android
```js
{
  "livemode": false,
  "bankAccount": null,
  "card": {
    "address": {
      "addressPostalCode": null,
      "addressState": null,
      "addressLine2": null,
      "addressLine1": null,
      "addressCountry": null,
      "addressCity": null
    },
    "last4": "3220",
    "funding": "Credit",
    "expMonth": 12,
    "currency": null,
    "expYear": 2022,
    "brand": "Visa",
    "country": "IE"
  },
  "type": "Card",
  "created": 2142850408,
  "id": "tok_1IsTEbG7gfqYEkPOgsQSjrLP"
}
```